metaci/cumulusci/views.py

```diff
@@ -17,14 +16,21 @@
 from metaci.cumulusci.models import Org
 from metaci.cumulusci.models import ScratchOrgInstance
 from metaci.cumulusci.utils import get_connected_app
+from metaci.plan.models import PlanRepository
 
 
-@staff_member_required
+@login_required
```

Comment:
i'm pretty sure login_required sends people to the admin login screen, not through social/our templates. this is going to be more annoying with community users.